lib/chat_api/conversations.ex

```diff
@@ -152,6 +152,8 @@ defmodule ChatApi.Conversations do
 
   """
   def update_conversation(%Conversation{} = conversation, attrs) do
+    Helpers.send_conversation_state_update(conversation, attrs)
```

Comment:
let's call this separately after the `update_conversation` succeeds, rather than embedding it here within the method itself

(same for the methods below)